### PR TITLE
ign -> gz Migrate Examples : gz-gui

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/711
Related to: https://github.com/gazebosim/gz-gui/pull/401

### CODE REVIEW

File: src/Application.cc [hunk 1]
```diff
@@ -91,15 +91,15 @@ using namespace gui;
 Application::Application(int &_argc, char **_argv, const WindowType _type)
   : QApplication(_argc, _argv), dataPtr(new ApplicationPrivate)
 {
-  igndbg << "Initializing application." << std::endl;
+  gzdbg << "Initializing application." << std::endl;
```
Comment (chapulina) [inline src/Application.cc:94]: Oops, these can't go in yet

**Review by chapulina — CHANGES_REQUESTED** [review]
The moved files LGTM, but we have to revert the `gzdbg` changes

File: src/Application.cc (hunk 1 above)
Comment (methylDragon) [reply src/Application.cc:94]: Oops

Comment (methylDragon) [reply src/Application.cc]: Force pushed and now this commit is :boom:

**Review by chapulina — APPROVED** [review]

**Pull request merged**